charts/core/values.yaml

```diff
@@ -15,7 +15,23 @@ leastPrivilege: false
 global: # required for rancher authentication (https://<Rancher_URL>/)
   cattle:
     url:
-
+  azurebilling:
```

Comment:
We need to follow the naming structure Microsoft dictates for the CNAB

https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes?tabs=linux#update-the-helm-chart

Reply:
we need to change `azurebilling` to `azure`